persistence/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraConfig.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import scala.language.implicitConversions
+
+import akka.actor.ActorSystem
+import javax.inject.Inject
+import javax.inject.Provider
+import javax.inject.Singleton
+
+trait CassandraConfig {
+  def uris: Set[(String, String)]
+}
+
+object CassandraConfig {
```

Comment:
Actually, it was an oversight, that class shouldn't even be in the in the persistence project, since it has the Cassandra [tcp address hardcoded](https://github.com/lagom/lagom/pull/53/files#diff-5e2183aefb08773c1911ce0a15a3feedR31). The `CassadraConfig` is currently being used by two projects: `teskit` and `cassandra-registration`. Since neither of them depend on the other, I'll create a new project that contains this source and can be consumed by both `teskit` and `cassandra-registration`. Thanks for spotting it!